alf/networks/projection_networks.py

```diff
@@ -260,7 +260,7 @@ def __init__(self,
             max_std (float): Maximum value for standard deviation. If None, no
                 maximum is enforced.
             dist_squashing_transform (td.Transform):  A distribution Transform
-                which transform values to fall in (-1, 1). Default to `dist_utils.StableTanh()`
+                which transforms values into :math:`(-1, 1)`. Default to `dist_utils.StableTanh()`
```

Comment:
double quotes \`` for code; and some arguments above have incorrect format

Reply:
updated